contrib/mhchem/index.html

```diff
@@ -0,0 +1,752 @@
+<!DOCTYPE html>
+<!--To text this example, place mhchem.js and this file into the same folder
+    and run this file.
+  -->
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>mhchem test</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/katex.min.css" integrity="sha384-9tPv11A+glH/on/wEu99NVwDPwkMQESOocs/ZGXPoIiLE8MU/qkqUcZ3zzL+6DuH" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/katex.min.js" integrity="sha384-U8Vrjwb8fuHMt6ewaCy8uqeUXv4oitYACKdB0VziCerzt011iQ/0TqlSlv8MReCm" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/contrib/auto-render.min.js" integrity="sha384-aGfk5kvhIq5x1x5YdvCp4upKZYnA8ckafviDpmWEKp4afOZEqOli7gqSnh8I6enH" crossorigin="anonymous"></script>
```

Comment:
If we decide to maintain the extension at this repo, we can do
```html
<script src="/katex.js" type="text/javascript"></script>
<script src="/contrib/auto-render.js" type="text/javascript"></script>
```
and access the page in `http://localhost:7936/contrib/mhchem/index.html` with `npm start`.

Reply:
I think that make sense as this `index.html` is more about testing.  It would be nice to have something within `webpage` to demo this there.

I love that this contrib modules works with the auto-render module.